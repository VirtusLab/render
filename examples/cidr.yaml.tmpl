{{ cidrHost 16 "10.12.127.0/20" }}
{{ cidrHost 268 "10.12.127.0/20" }}
{{ "fd00:fd12:3456:7890:00a2::/72" | cidrHost 34 }}

{{ cidrHostEnd 0 "10.12.127.0/20" }}
{{ cidrHostEnd 268 "10.12.127.0/20" }}
{{ cidrHostEnd 34 "fd00:fd12:3456:7890:00a2::/72" }}
{{ "10.12.127.0/20" | cidrHostEnd 16 }}

{{ cidrNetmask "10.0.0.0/12" }}

{{ index (cidrSubnets 2 "10.0.0.0/16") 0 }}
{{ index ("10.0.0.0/16" | cidrSubnets 2) 1 }}
{{ index (cidrSubnets 3 "10.0.0.0/16") 5 }}

{{ range $k, $v := cidrSubnetSizes 4 4 8 4 "10.1.0.0/16" }}
{{ $k }} {{ $v }}{{ end }}
{{ range $k, $v := cidrSubnetSizes 16 16 16 32 "fd00:fd12:3456:7890::/56" }}
{{ $k }} {{ $v }}{{ end }}
