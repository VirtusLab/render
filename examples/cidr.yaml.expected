10.12.112.16
10.12.113.12
fd00:fd12:3456:7890::22

10.12.127.254
10.12.126.242
fd00:fd12:3456:7890:ff:ffff:ffff:ffdc
10.12.127.238

255.240.0.0

10.0.0.0/18
10.0.64.0/18
10.0.160.0/19


0 10.1.0.0/20
1 10.1.16.0/20
2 10.1.32.0/24
3 10.1.48.0/20

0 fd00:fd12:3456:7800::/72
1 fd00:fd12:3456:7800:100::/72
2 fd00:fd12:3456:7800:200::/72
3 fd00:fd12:3456:7800:300::/88
